### DISCUSSION TITLE
Publishing builded image

### DISCUSSION THREAD

**Discussion by Cloud-Kid** [lbr38/repomanager#113] [Ideas]
Hi,
Your project looks promising ! I really wanted to take a look and maybe implement it in my company environment. But because the image is built at the starting time and not pulled from a repo like docker hub, some of the URLs used for the build are blocked in my environment.
Do you plan to release a docker image in the future instead ? 
Keep going, it looks fantastic !
Cloudkid

**Comment by lbr38** [comment]
Hello Cloudkid,
Thank you for your interest in the project and for your encouragement!

This is an idea I had thought about but never implemented or tested.
I'll see what I can do to help you, and I'll let you know quickly.

**Comment by lbr38** [comment]
Hi

Please check out the docker image installation instructions from: https://hub.docker.com/r/lbr38/repomanager

And tell me if something isn't clear, I'll do what is necessary to adapt the procedure.

**Reply by Cloud-Kid** [reply]
Hey,

I saw that you made edits to automate image building this week-end, I already tried it at home ! Thanks for your work I really apprciate it.
I deployed it in pre-production in my company today. We need to learn how to handle your software properly first.

Let me know if I can do anything to help you with this project. I really haven't found anything similar (Any open-source project to handle repos)

I really think your project can get more traction, maybe you can publish your work at r/homelab or r/sysadmin community on Reddit if thats something that may interest you.

Thanks a lot, I will keep a close eye on your updates :D 

Cloudkid

**Reply by lbr38** [reply]
Hi

Thanks, glad you like it!

I tried to document as much as possible each possible action with some GIFs to illustrate. Some GIFs may be outdated because I've made quite a few UI changes lately and haven't had time to redo the GIFs yet. Let me know if you have any problems.

I don't have a Reddit account unfortunately..

**Reply by Cloud-Kid** [reply]
The documentation is on point with the GIFs it's really understandable, I may post some more improvements, issues or ideas if I find some ! 

FYI I talked about your software on Reddit : https://www.reddit.com/r/sysadmin/comments/17c7zzj/i_was_looking_for_a_software_to_manage_linux/

**Reply by lbr38** [reply]
Thank you for your support and help, I really appreciate it :)
